### ISSUE TITLE
[BUG/Feature] Add a confirmation alert box when clicking the "Delete" button for subdomains.

### ISSUE THREAD

**Issue by nickchomey** [litespeedtech/cyberpanel#840]
There REALLY needs to be a confirmation button when you click the button for deleting subdomains. 

![image](https://user-images.githubusercontent.com/88559987/161153331-c4b295a4-339e-47bc-b989-26820ec1cac0.png)

There is one for the main Websites -> Delete Website feature. 

![image](https://user-images.githubusercontent.com/88559987/161153401-a3f7913e-b88d-468a-aaa9-da06a922c07c.png)

**Comment by master3395** [comment]
I second this change.
And deleting from inside the domain does not really delete the whole thing, it removes the connection out, but it does not remove files and it keeps making SSL errors, related to the one below, just so you are aware.
https://community.cyberpanel.net/t/deleted-sub-domain-not-fully-removed/33208/3

**Comment by nickchomey** [comment]
Yeah, I've noticed that files are only removed sporadically - not just for subdomains but even actual domains.

**Comment by master3395** [comment]
Thanks for the report. This issue is older and may not apply to current releases.

Please confirm:
1) CyberPanel version + build (please retest on the latest public release **2.4.9**)
2) OS name and version (whatever you run, e.g. `cat /etc/os-release`)
3) Exact steps still failing on a current install
4) Relevant logs (no passwords)

Note: the latest public release is **2.4.9**. Development branches are not recommended for reproducing bugs.

If we do not hear back within 30 days, this issue may be closed as stale. You can reopen with fresh details anytime.
